runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcSceneOverlay.java

```diff
@@ -176,11 +176,14 @@ private void renderNpcOverlay(Graphics2D graphics, NPC actor, Color color)
 		if (config.drawNames())
 		{
 			String npcName = Text.removeTags(actor.getName());
-			Point textLocation = actor.getCanvasTextLocation(graphics, npcName, actor.getLogicalHeight() + 40);
-
-			if (textLocation != null)
+			if (npcName != null)
```

Comment:
This should check for actor.getName not being null, before sending it to removeTags

Reply:
good idea

Reply:
Any update?